### DISCUSSION TITLE
Logo idea?

### DISCUSSION THREAD

**Discussion by ThatOneCalculator** [idiotWu/smooth-scrollbar#461] [Show and tell]
[In my project, we have all the logos of the packages we use as little boxes](https://github.com/Domterion/zer0bin#technologies-used), and I noticed this project didn't have one so I made this from the scroll icon + the gradient on the website! Hope you enjoy :)

![SmoothScrollJS](https://user-images.githubusercontent.com/44733677/158547006-29bf26f9-5973-4671-b360-63e1e99156ff.svg)

**Comment by dolphin-wood** [comment]
Wow! Great thanks! The logo looks super nice!

I'll add this to README soon! 
(As for the website, I'd like to add the logo in v9 since we are planning a big update 😆.)

**Reply by ThatOneCalculator** [reply]
Awesome! Glad you like it :heart:

**Reply by dolphin-wood** [reply]
@ThatOneCalculator and of course you are welcome to send a PR with the updated README.md if you have time :). Many thanks and a huge hug again 🤗.

**Reply by ThatOneCalculator** [reply]
#462 :smiley:
